frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/general-tab/simulated-region-overview-general-tab.component.html

```diff
@@ -1,13 +1,157 @@
-<div class="form-group pb-3">
-    <label class="form-label">Name</label>
-    <input
-        #nameInput="ngModel"
-        [ngModel]="simulatedRegion.name"
-        (appSaveOnTyping)="renameSimulatedRegion($event)"
-        required
-        type="text"
-        class="form-control"
-        data-cy="simulatedRegionPopupNameInput"
-    />
-    <app-display-validation [ngModelInput]="nameInput" />
+<div class="h-100 mh-100 overflow-auto">
+    <div class="form-group pb-3">
+        <label class="form-label">Name</label>
+        <input
+            #nameInput="ngModel"
+            [ngModel]="simulatedRegion.name"
+            (appSaveOnTyping)="renameSimulatedRegion($event)"
+            required
+            type="text"
+            class="form-control"
+            data-cy="simulatedRegionPopupNameInput"
+        />
+        <app-display-validation [ngModelInput]="nameInput" />
+    </div>
+
+    <h6>Aktuelle Daten</h6>
+
+    <table class="table">
+        <tbody>
+            <tr
+                (click)="patientsCollapsed = !patientsCollapsed"
+                style="cursor: pointer"
+            >
+                <th scope="rowgroup" class="w-75">
+                    <i
+                        *ngIf="patientsCollapsed; else patientsExpanded"
+                        class="bi bi-caret-right-fill"
+                    ></i>
+                    <ng-template #patientsExpanded
+                        ><i class="bi bi-caret-down-fill"></i
+                    ></ng-template>
+                    Patienten
+                </th>
+                <th class="w-25">{{ (patients.all$ | async)?.length }}</th>
+            </tr>
+            <tr #collapse="ngbCollapse" [(ngbCollapse)]="patientsCollapsed">
+                <td class="ps-4">
+                    <span
+                        class="badge rounded-pill"
+                        style="background-color: red"
+                        >SK I</span
+                    >
```

Comment:
There is a patient status badge component that you could use here

Reply:
Done in https://github.com/hpi-sam/digital-fuesim-manv/pull/721/commits/84701fed2df455dbb751c88073c181bb82f0d189